script.sql

```diff
@@ -0,0 +1,162 @@
+
+
+-- tables
+-- Table: app
+CREATE TABLE app (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(25) NOT NULL,
+    description text NOT NULL,
+    weight int NOT NULL,
+    create_date date NOT NULL,
+    is_free bool NOT NULL,
+    price decimal(8,2) NULL,
+    has_advert bool NOT NULL,
+    category_id int NOT NULL,
+    CONSTRAINT app_pk PRIMARY KEY (id)
+);
+
+-- Table: app_category
+CREATE TABLE app_category (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(30) NOT NULL,
+    description text NULL,
+    CONSTRAINT app_category_pk PRIMARY KEY (id)
+);
+
+-- Table: app_creating
+CREATE TABLE app_creating (
+    app_id int NOT NULL,
+    creater_id int NOT NULL,
+    CONSTRAINT app_creating_pk PRIMARY KEY (app_id,creater_id)
+);
+
+-- Table: card
+CREATE TABLE card (
+    number int NOT NULL,
+    bank_name varchar(25) NOT NULL,
+    user_id int NOT NULL,
+    CONSTRAINT card_pk PRIMARY KEY (number)
```

Comment:
Bad idea to use something as card number for your PK. [Here](https://stackoverflow.com/questions/31560264/is-it-practical-to-use-the-persons-id-card-as-a-primary-key) is a similar example why